components/Table/react.stories.js

```diff
@@ -125,7 +126,7 @@ storiesOf('Table', module)
       </tbody>
     </table>
   ))
-  .add('HTML', () => (
+  .add('Headerless Table', () => (
```

Comment:
This story should remain named as HTML. I meant for you to add your example in your Test story to this story. There are multiple tables in this story showing the table using only HTML. You can add a similar table without a header as an example that matches the established pattern.